docs/guides/hooks-list.rst

```diff
@@ -585,6 +585,14 @@ Other
 	"container_guid" (int) are provided. An empty entity value generally means the form is to
 	create a new object.
 
+**elgg_channels:send, <channel_name>**
```

Comment:
This is a server side hook we are looking at, so if I want to use a WS clinet for all the likes, I would have to either register with `all` keyword and sniff, or iterate from 1 to the latest id in the database to register all hook handlers :)
Perhaps using GUIDs in the API would make sense. `createChannel('likes', $guid)`. If no guid, just use the channel prefix/suffix.